x-pack/plugin/esql/qa/testFixtures/src/main/java/org/elasticsearch/xpack/esql/EsqlTestUtils.java

```diff
@@ -827,6 +829,7 @@ public static Literal randomLiteral(DataType type) {
                     throw new UncheckedIOException(e);
                 }
             }
+            case DENSE_VECTOR -> Arrays.asList(randomArray(10, 10, i -> new Double[10], ESTestCase::randomDouble));
```

Comment:
Should we actually `randomFloat()` and convert the value to double?
The rationale being that if we just use random double values, we might actually end up with something that can't be represented as a float and can't be used to actually index a `dense_vector` value.

Reply:
Makes sense - Done in e8878a04db0264c7669fbd7daf12e0224cf4ed18